otio_openassetio/operations/openassetio_media_linker.py

```diff
@@ -36,19 +40,33 @@ def link_media_reference(in_clip, media_linker_argument_map):
     if not isinstance(mr, otio.schema.ExternalReference):
         return
 
+    # The OpenAssetIO session is stateful to allow stable resolution
+    # over time, however this function isn't, so we track a shared
+    # session based on the arguments to the media linker.
     session_state = _sessionState(media_linker_argument_map)
     manager = session_state.session.currentManager()
 
     if not extract_one(manager.isEntityReference([mr.target_url])):
         return
 
+    # Update the locale with more information about this call
+    # This is perhaps more illustrative than useful at this point.
     context = session_state.context
-    context.locale.clipName = in_clip.name
-
-    mr.target_url = extract_one(
-        manager.resolveEntityReference([mr.target_url], context)
+    ClipTrait(context.locale).setName(in_clip.name)
+
+    # In this simple implementation, we only need the URL to the media,
+    # so we use the LocatableContentTrait directly to avoid the manager
```

Comment:
[consider] Might be worth adding a note "as opposed to all the traits of a particular specification"?

Reply:
Extended [here](https://github.com/TheFoundryVisionmongers/otio-openassetio/pull/5/commits/8b3a01d818135f3619f595e6bcb461d77ee63729).